Dockerfile

```diff
@@ -0,0 +1,23 @@
+# ---------------------- Builder image ----------------------
+
+FROM node:12-slim AS builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm i
+
+RUN npm run build
+
+RUN rm -rf src/
+
+# ---------------------- Runtime image ----------------------
+
+FROM node:12-slim
```

Comment:
think there is a hard requirement on node 10.
Should we be using our tini images?